src/SDKs/CognitiveServices/dataPlane/Language/LUIS-Programmatic/Customizations/ApiKeyServiceClientCredentials.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Microsoft.Azure.CognitiveServices.Language.LUIS.Programmatic
+{
+    using System;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Rest;
+
+    /// <summary>
+    /// Allows authentication to the API using a basic apiKey mechanism
+    /// </summary>
+    public class ApiKeyServiceClientCredentials : ServiceClientCredentials
+    {
+        private readonly string subscriptionKey;
+
+        /// <summary>
+        /// Creates a new instance of the ApiKeyServiceClientCredentails class
```

Comment:
Documentation standards require punctuation and sentences here and throughout the file.